src/test/java/org/opensearch/security/securityconf/SecurityRolesPermissionsTest.java

```diff
@@ -72,6 +76,17 @@ static String restAdminApiRoleName(final String endpoint) {
         return String.format("security_rest_api_%s_only", endpoint);
     }
 
+    static final Map<String, ObjectNode> HAS_SYSTEM_INDEX_PERMISSION = ImmutableMap.<String, ObjectNode>builder()
```

Comment:
You can use `Map.of(key, value, key, value)` instead of `ImmutableMap`

Reply:
@willyborankin 

Thank you for your comments.
I fixed it here: https://github.com/opensearch-project/security/pull/4430/commits/68b1370aa6863074f871cb3d26e6440c4b02b4a7.